Source/WebKit/_WebKit_SwiftUI/Implementation/WebViewRepresentable.swift

```diff
@@ -71,6 +71,9 @@ struct WebViewRepresentable {
         }
 #endif
 
+        let scrollInputBehavior = environment.webViewScrollInputBehaviorContext
```

Comment:
this should be if let, since otherwise the implementation of webView.configureScrollInputBehavior will crash if its nil